apis/v1alpha1/httproute_types.go

```diff
@@ -332,6 +336,8 @@ type HTTPRouteFilter struct {
 	// Filter-specific configuration definitions for core and extended filters
 
 	RequestHeader *HTTPRequestHeaderConfig `json:"requestHeader" protobuf:"bytes,3,opt,name=requestHeader"`
+
+	RequestMirror *HTTPRequestMirrorFilter `json:"requestMirror" protobuf:"bytes,4,opt,name=requestMirror"`
```

Comment:
Nit: RequestMirror and RequestHeader both make sense in isolation but seem a bit confusing when side by side like this. Maybe it would be clearer to have field names like `modifyHeaders` (or `modifyRequestHeaders`) and `mirrorRequest`.

Reply:
Great point. Can I change this in a follow up PR? (I've taken a note of this)

Reply:
👍 I'm fine with this as a follow up. Mind creating an issue to track that?

Reply:
https://github.com/kubernetes-sigs/service-apis/issues/307